variables.tf

```diff
@@ -47,7 +47,7 @@ variable "subnet_delegation" {
   nullable    = false
 }
 
-variable "subnet_enforce_private_link_endpoint_network_policies" {
+variable "subnet_enforce_private_endpoint_network_policies_enabled" {
```

Comment:
The new argument's name is [`private_endpoint_network_policies_enabled`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/subnet#private_endpoint_network_policies_enabled), could we rename this variable to `subnet_private_endpoint_network_policies_enabled`?

Reply:
done here